src/Microsoft.Identity.Client/HttpEvent.cs

```diff
@@ -30,35 +30,42 @@ namespace Microsoft.Identity.Client
 {
     internal class HttpEvent : EventBase
     {
+        public const string ConstHttpPath = "http_path";
```

Comment:
We will. We did that in the main logic PR #314. Once that one got merged, the change will be picked up here.